packages/openneuro-app/src/scripts/pages/image-attribution.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import type { ReactElement } from "react"
+import { Attribution } from "../common/content/image-attribution"
+import Helmet from "react-helmet"
+import { frontPage } from "./front-page/front-page-content"
+import styled from "@emotion/styled"
+
+const ImageAttributionStyle = styled.div`
+  background: white;
+
+  .container {
+    max-width: 60em;
+  }
+`
+
+export function ImageAttribution(): ReactElement {
+  return (
+    <ImageAttributionStyle>
+      <Helmet>
+        <title>Terms and Conditions - {frontPage.pageTitle}</title>
```

Comment:
Would you mind adding a screenshot of this page? Also:

```suggestion
        <title>Image Attribution - {frontPage.pageTitle}</title>
```